.github/workflows/_buildpacks-release.yml

```
@@ -212,29 +224,29 @@ jobs:
         with:
           repository: heroku/builder
           path: ./builder
+          # The token is used to set the local git config which is used to run authenticated git commands
+          token: ${{ steps.generate-token.outputs.app_token }}
+
+      # This will ensure commits made from this workflow are attributed to the bot user
+      - name: Configure git
+        run: |
+          git config --global user.name ${{ env.BOT_USER }}
+          git config --global user.email ${{ env.BOT_EMAIL }}
 
       - name: Install crane
         uses: buildpacks/github-actions/setup-tools@v5.3.0
 
-      - name: Update Builder
-        uses: heroku/languages-github-actions/.github/actions/update-builder@main
-        with:
-          repository_path: ./buildpacks
-          builder_repository_path: ./builder
-          builders: |
-            builder-22
-            buildpacks-20
+      - name: Install Languages CLI
+        uses: heroku/languages-github-actions/.github/actions/install-languages-cli@main
 
-      - name: Get token for GitHub application (Linguist)
-        uses: heroku/use-app-token-action@main
-        id: generate-token
-        with:
-          app_id: ${{ inputs.app_id }}
-          private_key: ${{ secrets.app_private_key }}
+      - name: Update Builder
+        # The dry run check is performed here because the update process requires a published to exist in oder to calculate a digest with `crane`
+        # or it fails. Adding the check here means no files will be modified and no PR will be created for the PR-related steps that follow.
```

(there were a few missing words/typos, and I have shortened slightly)

```suggestion
        # The dry run check is performed here because the update process requires a published
        # image to exist in order to calculate a digest with `crane`. Adding the check here
        # means no files will be modified and so no PR will be created later.
```